docs/sources/operators-guide/configuring/reference-configuration-parameters/index.md

```diff
@@ -287,6 +297,16 @@ grpc_tls_config:
   # CLI flag: -server.grpc-tls-ca-path
   [client_ca_file: <string> | default = ""]
 
+  [cipher_suites: <list of int> | default = ]
+
+  [curve_preferences: <list of int> | default = ]
+
+  [min_version: <int> | default = ]
+
+  [max_version: <int> | default = ]
+
+  [prefer_server_cipher_suites: <boolean> | default = ]
```

Comment:
We shouldn't have these here, unless they're fully supported. By fully supported I mean you can't set them via CLI flag (missing mapping in weaveworks/common) and "list of int" doesn't really provide a good user experience (what are the supported values? what does it do? ...).

I would be more keen to hide them, and not let pop up in the doc at all for now (also because the wouldn't be supported by the other TLS configs we have).

Reply:
Thanks @pracucci - will try to hide tehm.

Reply:
I've removed these config parameters from weaveworks/common.